tests/test_csv.py

```diff
@@ -3,15 +3,15 @@
 import json
 from pprint import pprint
 
-from csv_detective.explore_csv import routine
+from explore_csv import routine
 
 
 def sort_keys(dicto):
     return dict(sorted(dicto.items(), key=lambda x: x[0]))
 
 
 def test_old_detection():
-    file_path = './annuaire-de-leducation.csv'
+    file_path = '../../mlearnable-datasets-detective/data/csv/annuaire-de-leducation.csv'
```

Comment:
To change with a filepath on repo. Here : "./annuaire-de-leducation.csv'

Reply:
done